pkg/apis/maistra/v1/servicemeshmemberroll_types.go

```diff
@@ -68,6 +68,12 @@ const (
 	ConditionReasonConfigured ServiceMeshMemberRollConditionReason = "Configured"
 	// ConditionReasonNamespaceMissing indicates that one of the namespaces to configure does not exist
 	ConditionReasonNamespaceMissing ServiceMeshMemberRollConditionReason = "ErrNamespaceMissing"
+	// ConditionReasonSMCPMissing indicates that the ServiceMeshControlPlane resource does not exist
+	ConditionReasonSMCPMissing ServiceMeshMemberRollConditionReason = "ErrSMCPMissing"
+	// ConditionReasonMultipleSMCP indicates that multiple ServiceMeshControlPlane resources exist in the namespace
+	ConditionReasonMultipleSMCP ServiceMeshMemberRollConditionReason = "ErrMultipleSMCPs"
+	// ConditionReasonSMCPNotReconciled indicates that reconciliation of the SMMR was skipped because the SMCP has not been reconciled
+	ConditionReasonSMCPNotReconciled ServiceMeshMemberRollConditionReason = "ErrSMCPNotReconciled"
 )
```

Comment:
Is this really an `Err`? I don't think it's an error if the SMCP not reconciled. It just means reconciliation did not occur because the control plane was in an indeterminate state.  Maybe `SMCPReconciling`?

Reply:
Good point. I'll change it to `SMCPReconciling`